### ISSUE TITLE
`with-credentials` is not respected for skybox-image

### ISSUE THREAD

**Issue by robertsLando** [google/model-viewer#4083]
### Description

I have `with-credentials` attribute set and it works well both for model and poster image but seems it's not respected for skybox-image as it gives me a 403 error and `Cookie` header is not set.

### Version

- model-viewer: v2.1.1

### Browser Affected

- [X] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [X] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by robertsLando** [comment]
Hi @elalish ,

I'm sorry but seems the issue is still there after 3.0.1 version. By debugging it seems an error on `three.module.js`:

- `loadEquirect` is called. `hdrLoader` has `withCredentials` set to `true` 🆗 
- `loader.load` is called, this calls `three.model.js` `load` function that does this:

```js
load( url, onLoad, onProgress, onError ) {

		const texture = new Texture();

		const loader = new ImageLoader( this.manager );
		loader.setCrossOrigin( this.crossOrigin );
		loader.setPath( this.path );
		loader.setWithCredentials(this.withCredentials); // <---- THIS iS MISSING

		loader.load( url, function ( image ) {

			texture.image = image;
			texture.needsUpdate = true;

			if ( onLoad !== undefined ) {

				onLoad( texture );

			}

		}, onProgress, onError );

		return texture;

	}
```

As you can see there a new ImageLoader instance is created and there is nothing that sets the `withCredentials` prop that is correctly set to true in the `TextureLoader` but not in the new intance created

**Comment by robertsLando** [comment]
I opened a PR on three npm package: https://github.com/mrdoob/three.js/pull/25514

**Comment by robertsLando** [comment]
@elalish seems that the issue is here but I dunno how to help, see discussion on above pr link

**Comment by robertsLando** [comment]
TLDR:

https://github.com/mrdoob/three.js/pull/25514#discussion_r1108210424

**Comment by elalish** [comment]
Thanks, it sounds like there's not much I can do at the moment.
